guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc

```diff
@@ -13,19 +13,29 @@ To enable the Ansible plug-in in {Project}, enter the following command:
 
 [options="nowrap" subs="+quotes,attributes"]
 ----
-  # {foreman-installer} --enable-foreman-plugin-ansible \
-  --enable-foreman-proxy-plugin-ansible
+# {foreman-installer} --enable-foreman-plugin-ansible \
+--enable-foreman-proxy-plugin-ansible
 ----
-
 endif::[]
 
+[id="Ansible-paths_{context}"]
+.Ansible paths
+
+{Project} imports Ansible roles and variables from paths based on configuration in `/etc/ansible/ansible.cfg`.
+{Project} then runs imported roles from paths based on configuration in `/etc/foreman-proxy/ansible.cfg`.
+In both cases, {Project} reads the paths from `roles_path` and `collections_path` directives.
+You should keep these two cfg files in sync, otherwise you might import roles that cannot be run or you will not see roles you can run.
```

Comment:
IBM style guide recommends not using "should". Can you please reword this if possible?

I guess saying "keep them in sync or bad things happen" is OK.